libs/libcommon/src/libcommon/parquet_utils.py

```diff
@@ -30,6 +31,18 @@ class FileSystemError(Exception):
     pass
 
 
+def _clean_mongo_objects(obj: Any) -> Any:
```

Comment:
maybe move this to the root function in simple_cache.py that gets the content out of the MongoDB?